templates/patterns/facts_figures/facts_figures.ui_patterns.yml

```diff
@@ -0,0 +1,40 @@
+facts_figures:
+  label: "Facts and figures"
+  description: "The facts and figures component is used to deliver numerical representations of facts that are easier to portray visually through the use of statistics."
+  fields:
+    items:
+      type: "array"
+      label: "Facts"
+      description: "List of facts"
+      preview:
+        - icon: "branded--instagram"
+          value: "00.0 million"
+          title: "Lorem ipsum"
+          description: "Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero."
+        - icon: "branded--twitter"
+          value: "00.0 million"
+          title: "Sed hendrerit"
+          description: "Turpis varius congue venenatis, erat dui feugiat felis."
+        - icon: "general--calendar"
+          value: "00.0 million"
+          title: "Donec suscipit interdum augue, ac dapibus eros finibus a."
+          description: "Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem."
+        - icon: "general--euro"
+          value: "00.0 million"
+          title: "Aenean dapibus"
+          description: "Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum."
+        - icon: "notifications--success"
+          value: "00.0 million"
+          title: "Aliquam faucibus nulla eget eleifend"
+          description: "Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor."
+        - icon: "ui--external"
+          value: "00.0 million"
+          title: "Nam lacinia nisl eget diam mattis"
+          description: "Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur."
```

Comment:
We shouldn't use ECL names here, but our names, as ECL might change them one day. If that happens then we will change them in the `to_icon` filter only.

Let's use simple names here, like those we used on the paragraph, so something like:

```yml
      preview:
        - icon: "instagram"
          value: "00.0 million"
          title: "Lorem ipsum"
          description: "Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero."
        - icon: "twitter"
          value: "00.0 million"
          title: "Sed hendrerit"
          description: "Turpis varius congue venenatis, erat dui feugiat felis."
```